turbo/snapshotsync/freezeblocks/bor_snapshots.go

```diff
@@ -30,15 +28,57 @@ import (
 	"github.com/ledgerwatch/erigon/eth/ethconfig"
 	"github.com/ledgerwatch/erigon/polygon/heimdall"
 	"github.com/ledgerwatch/erigon/turbo/services"
+	"github.com/ledgerwatch/log/v3"
 )
 
+func (br *BlockRetire) dbHasEnoughDataForBorRetire(ctx context.Context) (bool, error) {
+	/*
```

Comment:
Why is this commented out?

Reply:
bor-events exist not in all blocks. don't know which check to add here yet.